lib/travis/api/v3/models/repository_migration.rb

```diff
@@ -11,7 +11,9 @@ def initialize(repository)
     end
 
     def migrate!
-      if !Travis::Features.owner_active?(:allow_migration, repository.owner)
+      if !(Travis::Features.feature_active?(:allow_merge_globally) &&
```

Comment:
These two calls happen multiple times in the code. At least within v3, we could have this method called on a repository object as `repo.allow_migration?` where the method would be:
```
def allow_migration?
   Travis::Features.feature_active?(:allow_merge_globally) &&
     Travis::Features.owner_active?(:allow_migration, repository.owner)
end
```

In this model it can be used as
```
def migrate!
  unless repository.allow_migration?
      raise...
   end
end
```

And in the renderer it can be used as `model.allow_migration?` instead of the method `allow_migration`

Reply:
Sounds reasonable